### ISSUE TITLE
Server Error (500) from Web Shell or Web Desktop buttons

### ISSUE THREAD

**Issue by calvinmclean** [cyverse/atmosphere-docker#1]
When attempting to use one of the "Open Web Shell" or "Open Web Desktop" buttons, the result is a "Server Error (500)" on `https://localhost/web_desktop`.

Using the Network tab of dev tools, I clicked "Edit & Resend", removed one of the POST parameters, and resent. The result was a 400 page that said, "POST does not contain the required data", which is an error message from Troposphere's `web_desktop.py`.

If I manually enter the URL `https://localhost/api/v2/web_tokens/e5327c1e-5622-4a1b-baf8-3eab9f1f1038?client=guacamole&protocol=ssh`, which is what Troposphere is supposed to create, I get a returned `token_url` that is a valid Guacamole URL.

Also, from Troposphere logs:
```
ConnectionError: HTTPSConnectionPool(host='localhost', port=443): Max retries exceeded with url: /api/v2/web_tokens/e5327c1e-5622-4a1b-baf8-3eab9f1f1038?client=guacamole&protocol=ssh (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f0a081d0210>: Failed to establish a new connection: [Errno 111] Connection refused',))
```

This leads me to believe Troposphere is unable to reach the Atmosphere API V2.

**Comment by calvinmclean** [comment]
The Troposphere log entry shows that Troposphere creates the link but cannot reach `https://localhost/api/v2`. It cannot access `https://atmosphere/api/v2` or `https://nginx/api/v2`

**Comment by calvinmclean** [comment]
`curl "https://nginx/api/v2/web_tokens/b95cc042-faac-4620-a0d7-dd2dc2373a66?client=guacamole&protocol=ssh"` from Troposphere container:
```
curl: (51) SSL: no alternative certificate subject name matches target host name 'nginx'
```

If I use `http` instead:
```
<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.4.6 (Ubuntu)</center>
</body>
</html>
```

So I have to use nginx as the API_V2_HOST and make the SSL work with `nginx` hostname or disable HTTPS

**Comment by calvinmclean** [comment]
Fixed by 0a844397c8c389582b0f74b469df610edc2f990a
